engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -310,4 +342,17 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId, workflowDescript
         Success(WorkflowExecutionDiff(Map(collector -> ExecutionStatus.Starting)))
     }
   }
+
+  private def pushOutputsToMetadataService(data: WorkflowExecutionActorData): Future[Unit] = {
+    import workflow._
+    val keyValues = data.outputStore.store.flatMap {
+      case (key, value) => value map (entry => s"${key.call.fullyQualifiedName}.${entry.name}" -> entry.wdlValue.map(_.toWdlString).getOrElse("NA"))
+    }
+    val metadataEventMsgs = keyValues map{ case (k,v) => MetadataEvent(MetadataKey(workflowId, None, s"${WorkflowMetadataKeys.Outputs}.$k"), MetadataValue(v))}
```

Comment:
not `$` I'd say because we've used for scatter placeholder I think or logs. I'm pretty sure something uses it anyway.